src/Resources/contao/dca/tl_cfg_tag.php

```diff
@@ -92,6 +99,15 @@
             ],
             'sql' => ['type' => 'string', 'length' => 64, 'default' => ''],
         ],
+        'alias' => [
+            'label'         => &$GLOBALS['TL_LANG']['tl_cfg_tag']['alias'],
+            'exclude'       => true,
+            'search'        => true,
+            'inputType'     => 'text',
+            'eval'          => ['rgxp' => 'alias', 'unique' => true, 'maxlength' => 128, 'tl_class' => 'w50'],
+            'save_callback' => [['codefog_tags.listener.data_container.tag', 'generateAlias']],
+            'sql'           => "varchar(128) COLLATE utf8_bin NOT NULL default ''",
```

Comment:
Please adjust the SQL format just like it is for the other fields.

Reply:
Of course, I can do that, but at the moment I have no idea how to define the collation:

I tried ```'collation' => 'utf8_bin'``` and ```'collate' => 'utf8_bin'```. None of them worked out :-(